installers/launcher-config-mac.yaml

```diff
@@ -22,6 +22,7 @@ nodeArgs:
 - ./wallet-topology.yaml
 nodeDbPath: $HOME/Library/Application Support/Daedalus/DB-1.0
 nodeLogConfig: log-config-prod.yaml
+nodeLogPath: $HOME/Library/Application Support/Daedalus/Logs/cardano-node.log
```

Comment:
Actually it's enough to revert it only on Windows. Mac version works fine.

Reply:
Let's be conservative